src/Raven.Server/ServerWide/LocalEndpointClient.cs

```diff
@@ -88,7 +89,16 @@ private static void ThrowHttpRequestException(RouteInformation route, int status
         public async Task<BlittableJsonReaderObject> InvokeAndReadObjectAsync(RouteInformation route, JsonOperationContext context, Dictionary<string, StringValues> parameters = null)
         {
             var response = await InvokeAsync(route, parameters);
-            return context.ReadForMemory(response.Body, $"read/local endpoint/{route.Path}");
+
+            try
+            {
+                return context.ReadForMemory(response.Body, $"read/local endpoint/{route.Path}");
+            }
+            catch (InvalidStartOfObjectException e)
```

Comment:
Why catch just this exception, should do that for all exceptions.
Also, what happens if the endpoint fails for any reason? Do we fail the entire process?